packages/concerto-cto/lib/parser.pegjs

```diff
@@ -917,12 +917,29 @@ DecoratorIdentifier =
           ...buildRange(location())
       }
   }
+  
```

Comment:
You'll have to run the codegen command to generate the parser expressions for this new grammar.